static/katex.less

```diff
@@ -367,10 +368,16 @@
         right: 0;
     }
 
-    .rlap > .inner {
+    .rlap > .inner,
+    .clap > .inner {
         left: 0;
     }
 
+    .clap > .inner  > span {
+        margin-left: -50%;
+        margin-right: 50%;
+    }
```

Comment:
I feel like there might be an easier way to center things.  I'll check out the branch and experiment.

Reply:
I never got around to experimenting.  This does the trick and anything else would be marginally smaller so I'm going to merge this as is.